domains/primitives/runtime/src/lib.rs

```diff
@@ -174,6 +174,12 @@ pub struct CheckExtrinsicsValidityError {
 pub const CHECK_EXTRINSICS_AND_DO_PRE_DISPATCH_METHOD_NAME: &str =
     "DomainCoreApi_check_extrinsics_and_do_pre_dispatch";
 
+#[derive(Clone, Debug, Default, Decode, Encode, Eq, PartialEq, TypeInfo)]
+pub struct DomainBlockReward<Balance> {
```

Comment:
I'm thinking of renaming this struct to `DomainFees` and the `pallet-operator-rewards` to `pallet-domain-fees`, also `ER::total_rewards` to `ER::total_fees` as @dariolina suggested, WDYT @vedhavyas

Reply:
I would simply call it `BlockFees`  since we are planning to include `consensus_storage_fees` into this this data type and `Domain` seems redundant since its already part of the Domain ER